frontend/src/features/admin-form/responses/FeedbackPage/FeedbackPage.tsx

```diff
@@ -102,3 +228,135 @@ export const FeedbackPage = (): JSX.Element => {
     </Container>
   )
 }
+
+const getReviewInformationComponent = (
+  average: string | undefined,
+  count: number | undefined,
+): JSX.Element => {
+  return (
+    <Grid
+      gridTemplateColumns={{ base: 'auto', md: 'auto auto' }}
+      gridGap={{ base: '0.5rem', md: '1.5rem' }}
+      gridTemplateAreas={{
+        base: "'submissions' 'score'",
+        md: "'score submissions'",
+      }}
+    >
+      <Flex gridArea="score" flexDir="column">
+        <Text textStyle="caption-2" color="secondary.400">
+          Average Score
+        </Text>
+        <Text textStyle="display-2">
+          {average ? Number(average).toPrecision(2) : '-.--'}
+        </Text>
+      </Flex>
+      <Box gridArea="submissions" alignSelf="end">
+        <Text textStyle="h4" mb="0.5rem">
+          <Text as="span" color="primary.500">
+            {count}
+          </Text>
+          {simplur` ${[count || 0]}feedback submission[|s] to date`}
+        </Text>
+      </Box>
+    </Grid>
+  )
+}
+
+const getIssueInformationComponent = (
+  count: number | undefined,
+): JSX.Element => {
+  return (
+    <div>
+      <Box>
+        <Text textStyle="h4" mb="0.5rem">
+          <Text as="span" color="primary.500">
+            {count}
+          </Text>
+          {simplur` ${[count || 0]}issue[|s] to date`}
+          <Tooltip
+            label={`Feedback displayed here relates to form submission issues`}
+            placement="top"
+            textAlign="center"
+          >
+            <Icon as={BxsInfoCircle} aria-hidden marginX="0.5rem" />
+          </Tooltip>
+        </Text>
+      </Box>
+    </div>
```

Comment:
Unnecessary wrapping `div`. Also, you can use `<></>` next time if a wrapping div is really needed.
```suggestion
    <Box>
      <Text textStyle="h4" mb="0.5rem">
        <Text as="span" color="primary.500">
          {count}
        </Text>
        {simplur` ${[count || 0]}issue[|s] to date`}
        <Tooltip
          label={`Feedback displayed here relates to form submission issues`}
          placement="top"
          textAlign="center"
        >
          <Icon as={BxsInfoCircle} aria-hidden marginX="0.5rem" />
        </Tooltip>
      </Text>
    </Box>
```